src/Tribe/Attendees.php

```diff
@@ -790,34 +804,95 @@ public function maybe_generate_csv() {
 		 */
 		$items = apply_filters( 'tribe_events_tickets_attendees_csv_items', $items, $event_id );
 
-		if ( ! empty( $items ) ) {
-			$charset  = get_option( 'blog_charset' );
-			$filename = sanitize_file_name( $event->post_title . '-' . __( 'attendees', 'event-tickets' ) );
-
-			// Output headers so that the file is downloaded rather than displayed.
-			header( "Content-Type: text/csv; charset=$charset" );
-			header( "Content-Disposition: attachment; filename=$filename.csv" );
-
-			// Create the file pointer connected to the output stream.
-			$output = fopen( 'php://output', 'w' );
-
-			/**
-			 * Allow filtering the field delimiter used in the CSV export file.
-			 *
-			 * @since 5.1.3
-			 *
-			 * @param string $delimiter The field delimiter used in the CSV export file.
-			 */
-			$delimiter = apply_filters( 'tribe_tickets_attendees_csv_export_delimiter', ',' );
-
-			// Output the lines into the file.
-			foreach ( $items as $item ) {
-				fputcsv( $output, $item, $delimiter );
-			}
+		if ( empty( $items ) ) {
+			return;
+		}
 
-			fclose( $output );
-			exit;
+		$charset  = get_option( 'blog_charset' );
+		$filename = sanitize_file_name( $event->post_title . '-' . __( 'attendees', 'event-tickets' ) );
```

Comment:
Minor: this would be a good place to use a context translation string (`_x()`) to specify how "attendees" is used. It's possible that this could change how the word is translated, since it is appended to the event's title.